bootstrap/secret/secure.go

```diff
@@ -388,59 +388,59 @@ func (p *SecureProvider) HasSecret(path string) (bool, error) {
 	return true, nil
 }
 
-// ListSecretPaths returns a list of paths for the current service from an insecure/secure secret store.
-func (p *SecureProvider) ListSecretPaths() ([]string, error) {
+// ListSecretSecretNames returns a list of secretNames for the current service from an insecure/secure secret store.
+func (p *SecureProvider) ListSecretNames() ([]string, error) {
 
 	if p.secretClient == nil {
-		return nil, errors.New("can't get secret paths. Secure secret provider is not properly initialized")
+		return nil, errors.New("can't get secret secretNames. Secure secret provider is not properly initialized")
 	}
 
-	secureSecrets, err := p.secretClient.GetKeys("")
+	secureSecrets, err := p.secretClient.GetSecretNames("")
```

Comment:
No parameter needed here.
```suggestion
	secureSecrets, err := p.secretClient.GetSecretNames()
```